src/providers/RosterProvider/docs/RosterProvider.stories.mdx

```diff
@@ -0,0 +1,61 @@
+<Meta title="SDK Providers/RosterProvider" />
+
+# RosterProvider
+
+The `RosterProvider` provides the state of the roster.
+
+### State
+
+```javascript
+{
+ [AttendeeId: string]: {
+   chimeAttendeeId: string;
+   externalUserId?: string;
+   name?: string;
+ }
+}
+```
+You can access the state by using the [useRosterState](/?path=/docs/sdk-hooks-userosterstate--page) hook.
+
+## Usage
+
+If you are using `MeetingProvider`, the `RosterProvider` is rendered by default.
+
+```jsx
+import React from 'react';
+import {
+  MeetingProvider,
+  useRosterState,
+  Roster,
+  RosterGroup,
+  RosterAttendee
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => (
+  <MeetingProvider>
+    <MyChild />
+  </MeetingProvider>
+);
+
+const MyChild = () => {
+  const { roster } = useRosterState();
+  const rosterArray = Object.values(roster);
+
+  const attendeeItems = attendees.map(attendee => {
+    const { chimeAttendeeId, name } = attendee;
+    return (
+      <RosterAttendee key={chimeAttendeeId} attendeeId={chimeAttendeeId} />
+    );
+  });
+
+  return (
+    <Roster>
+      <RosterGroup>{attendeeItems}</RosterGroup>
+    </Roster>
+  );
```

Comment:
I guess this is up to the team, but I thought it would be good to show developers how the data from our RosterProvider would normally be used if you were building an app using our components library. You could technically use <ul> but this example gives a little more insight into how you could use the library to show the roster data, and encourages the developer to use other components of our library.